source/guide_mautic.rst

```diff
@@ -0,0 +1,381 @@
+    - Installing composer/installers (v1.12.0): Extracting archive
+  [...]
+  Generating autoload files
+  110 packages you are using are looking for funding.
+  Use the `composer fund` command to find out more!
+  Symfony recipes are disabled: "symfony/flex" not found in the root composer.json
+
+  Package {VENDOR/PACKAGE} is abandoned, you should avoid using it. Use {VENDOR/PACKAGE} instead. # Might come up multiple times. You can safely ignore it.
+
+  Loading composer repositories with package information
+  Updating dependencies
+  Nothing to modify in lock file
+  Installing dependencies from lock file (including require-dev)
+  Nothing to install, update or remove
+  Generating autoload files
+  Symfony recipes are disabled: "symfony/flex" not found in the root composer.json
+
+  Scaffolding files for mautic/core-lib:
+     - Copy [project-root]/.env.dist from assets/scaffold/files/.env.dist
+    [...]
+  No security vulnerability advisories found
+  Project configuration is disabled: "symfony/flex" not found in the root composer.json
+
+  Congratulations, you’ve installed the Mautic codebase
+  from the mautic/recommended-project template!
+
+  Next steps:
+    * Install Mautic
+    * Read the user guide
+    * Get support: https://www.mautic.org/support
+    * Get involved with the Mautic community:
+        https://www.mautic.org/getting-involved
+    * Remove the plugin that prints this message:
+        composer remove mautic/core-project-message
+    * Homepage: https://www.mautic.org/mautic-releases
+    * Support:
+      * user-docs: https://docs.mautic.org/en
+      * developer-docs: https://developer.mautic.org
+      * chat: https://www.mautic.org/slack
+  [isabell@stardust isabell]$
+
+
+Step 2: Create a symbolic link
+-------------------------------
+
+Create a symbolic link pointing to Mautic's ``docroot`` folder to let Apache serve Mautic's pages.
+You need to delete Apache's root directory (``/var/www/virtual/isabel/html``) for that.
+Make sure not to delete important files.
+
+.. code-block:: console
+
+  [isabell@stardust isabell]$ rm -f html/nocontent.html
+  [isabell@stardust isabell]$ rmdir html
+  [isabell@stardust isabell]$ ln -s /var/www/virtual/$USER/mautic/docroot/ /var/www/virtual/$USER/html
+
+This will result in the following structure:
+
+.. code-block:: console
+
+  [isabell@stardust isabell]$ tree -L 2
+  .
+  ├── html -> /var/www/virtual/isabell/mautic/docroot/
+  └── mautic
+      ├── app
+      ├── autoload.php
+      ├── bin
+      ├── composer.json
+      ├── composer.lock
+      ├── docroot
+      ├── Gruntfile.js
+      ├── package.json
+      ├── package-lock.json
+      ├── README.md
+      └── vendor
+
+
+Step 3: Check your Mautic installation
+----------------------------------
+
+Visit ``https://mautic.mysite.com`` (or ``https://isabell.uber.space``) in your browser.
+There might be a ``403 forbidden error``.
```

Comment:
there wont be an error when following the guide, you can remove the whole part about the 403 error